app/controllers/workarea/storefront/checkout/place_order_controller.decorator

```diff
@@ -0,0 +1,8 @@
+module Workarea
+  decorate Storefront::Checkout::PlaceOrderController, with: :cim do
+    def place_order
+      params.merge!(ip: request.remote_ip)
```

Comment:
This decoration will be unnecessary, see my comment on `payment.decorator` below.